lib/reverse_sentence.rb

```diff
@@ -1,6 +1,47 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(3n) == O(n) (n = number of elements in the input object; each loop has O(n) time complexity)
+# Space complexity: O(n) (n = number of elements in the input array)
+
+
 def reverse_sentence(my_sentence)
```

Comment:
👍

Reply:
> Comprehension questions?
> 
> Well done you hit the learning goals here Denise. Nice work.

I just fixed the problem with the comprehensive questions. Thank you, Chris!!